### ISSUE TITLE
Current status of QOSF website

### ISSUE THREAD

**Issue by s-peryt** [qosf/qosf.org#89]
I've recently found QOSF website and I find resources available there very useful for people interested in Quantum Computing. However, what I noticed is that website hasn't been updated for a while, especially when compared to discussions and commits available on GitHub. For example recent PR #87 is merged, but not present on the website. Similarly, issue #24 discusses rewrite to the manifesto, but I doubt anything with regards to that has been updated.

I'd like to participate in the development of QOSF but at the same time I don't think there is any way to achieve that. Can someone please provide some feedback regarding current idea/plan for the foundation?

Thanks!

**Comment by s-peryt** [comment]
@jalaln06 thanks for reviving this topic. Honestly, I completely forgot I've added this.
I see that there has been no movement with regards to bringing back this website. And also looks like all of the social media channels of QOSF are abandoned.

If I won't be beaten to it, or there wouldn't be any movement in this repository, I'm going to fork it into new reincarnation of the project. The plan is to create new domain (I guess this part is obvious) and also move away from foundation and mentoring part of the website. Focus on the various QC materials and resources available. To concentrate on knowledge sharing among people interested in the subject.

**Comment by MadhurimaNayak** [comment]
@s-peryt hey! Their [Linkedin](https://www.linkedin.com/company/qosf/) is pretty active and they regularly post updates there. Plus, they recently started Batch 9 of their mentoring program. So I think a good idea would be to update the mentorship part of the website instead of doing away with it.

**Comment by s-peryt** [comment]
Thanks for info @MadhurimaNayak . Their LinkedIn is not listed on the page footer, hence I didn't even know there is one.
Well, the question is then - what is the goal of QOSF. So far I've been treating it as a source for learning materials. And it seems (based also on what you pointed out) more on mentoring which is very limited IMHO.

**Comment by Kitsunp** [comment]
@s-peryt From what I've seen since I registered in the event is that QOSF is half divided from the original page, I mean that is still their reference page but for funds, almost everything goes into the projects, I mean to find people, mentors, mentoring participants, and more, in Mexico there was an event that referred to that, so not missing, but you can see that only focus on developing, also have been more active lately.

**Comment by s-peryt** [comment]
> almost everything goes into the projects

@Kitsunp sorry, I didn't get that. What projects? You mean the money from the funding goes into mentoring projects?
To be clear I don't look at this page for mentoring. When I first found it many years ago it was really good and comprehensive (back then) source of information for QC learning. And this is what I'd like to get revived.

Mentoring is nice, but this always felt like opportunity for very small number of selected people. And also strongly constrained by application windows. I remember I've applied to one of the first mentoring cohorts but because there weren't much mentors I didn't get pick. And IIRC back then it was once or twice a year event.

**Comment by Kitsunp** [comment]
@s-peryt as you said the mentoring almost takes all the budget, but still don't worry, currently a high ranking member within QOSF wants to revive the site, in Mexico for example another time, in the IPN he came to give us a talk and what he wants to achieve with the foundation or site, he talked about updating the site, adding more resources, allying with other groups to promote it in different places and more.

**Comment by s-peryt** [comment]
Ok, that's cool some "high ranking member within QOSF" is thinking of doing something. But also as you can see my ticket has almost 1 year. And last commit is also from around this time.
Frankly speaking if not for @jalaln06 I'd forgot I wanted to pick this up. But now when I have a bit more free time I'm willing to restart the learning part of the website. With the pace I've seen so far I have doubts any movement will happen soon.
It'll be just good to have any info from QOSF members (the one that can actually move content on the website forward - i.e. upload to qosf.org) what's the plan/roadmap.

**Comment by victor-onofre** [comment]
Hi @s-peryt  and @Kitsunp, thanks for your interest in helping us! We really appreciated. I think there is some confusion about what is happening in QOSF. I will put some updates about the foundation that I hope will clear the confusion: 

- The originals founders of QOSF left the organization for other commitments like 1.5 years ago. 
- The website and the different programs were left unattended for a while because of lack of people in the admin team.
- The mentorship program was running thanks to people still interesting in participating. 
- Me and Alberto (@MaldoAlberto) have been running the different cohorts of the mentorship and until recently we decided to take full control of the organization that's why your issue was ignored before. It is until a couple of months ago that we got full access to the website. 
- There is not a "high ranking member" of QOSF. We are doing this for free on our free time. 
- At the moment we are trying to reorganize the foundation but is a slow process given the limited time and small number in the admin team. But we thing the QOSF is great and little by little we will improve it. 
- There is no funding, we have zero money. Everything is done by free by people interested in the organization. The bounty from this issues are from the Unitary Fund. 
- The Unitary Hack is a great way to start getting interested in the website and try to find people that are willing to help. But it is true that we didn't explain all the tasks that we want to fix first clearly. Sorry about that! 
- I think one of the first things we need to clarify in the website is basically all the previous points. To update the community. 

Right now is late in my timezone but tomorrow I will clear the points of issues and assign more clear tasks. 

Happy to answer any questions you may have. 

Thanks!

**Comment by s-peryt** [comment]
Thank you @victor-onofre for explanation and providing more details. I'm glad that something will hopefully be moving forward.

As I said I am willing to allocate some time (I'm thinking few hours each week for now) in helping out in brining the site back-up. I might not have much practical experience in quantum computing but definitely I have experience in classical computers, especially in compilers, being compiler engineer for over 7 years now. Definitely this area is the most interesting for me, but after trying to find, read and understand various materials in wider QC space I noticed that probably some general overhaul would be helpful. Especially for people beginning in the area. And I'd like to help out in this.

**Comment by jalaln06** [comment]
Yes, I can agree with @s-peryt. Having started in QC about a year ago, I've looked over a lot of different materials, and a general overhaul is needed. I'm also willing to dedicate some of my time to it.

**Comment by victor-onofre** [comment]
@s-peryt, @jalaln06, thanks for your willingness to help on the project. Given that Alberto and I have a more research background, we don't have a lot of experience maintaining a website. You can really help us with some initial cleaning and modifications, and if you like working with us, we can add you as a maintainer to join the team.

For now we can start with some simple tasks:

- At the moment the only social media accounts active are LinkedIn and soon Youtube. Please update this on the website
- If we want this to be a more community driven project, we need to have some steps and rules for community contributions. I would like to see some instructions on how to modify the website for new people. Right now, it may not be clear.
- If you find some broken links, please try to fix them or delete them if they are unnecessary.
-  In the core team project we should update the section. I was thinking of calling the original founders the first generation of the admin team. We can update the information with bios of the current admin team. At one point, I would like to move from core members to a more decentralized community, but something that Alberto and I have realized is that it is necessary to put some faces for initial contact. Specialized right now that the foundation has been unattended for a while.
- The mentorship program is the only foundation activity that has been maintained and is still running. We should highlight this program on the website and update the section with information about previous cohorts.

Please let me know what you think about these tasks.

Thanks!

**Comment by MadhurimaNayak** [comment]
@s-peryt @jalaln06 Would you guys need any help updating the website? I'd also be up for it. I have recently started out in QC, but I could definitely help with updating the site and cleaning it up.

**Comment by jalaln06** [comment]
@victor-onofre 
I'd suggest to create some kind of community space for QOSF like a Discord server, where we can organize activities better. I think community rules could also be posted there. Most Quantum communities, and projects I saw use Discord servers. 

As when it comes to the tasks you mentioned. Sure, we can update the links and materials. I'm not sure if I will be able to do it soon, but definitely in a week. 

Speaking about admin names I'm not much into context, but I guess some kind of hall of fame for ex-team would be good.

@MadhurimaNayak Yes, sure! This is open-source.

**Comment by s-peryt** [comment]
@victor-onofre I'm fine with this plan. I'll take over fixing broken links (there is one PR from my already submitted - #88 ). As a part of this I'll also address #9 to order everything alphabetically. For all of this I've created #97

I'd suggest following things in general for now:
1. Put the list of your tasks as actionable GitHub issues. Already done this for few - #97, #98, #99, #100
2. I've noticed in some of the OS projects usage of "Projects" section of the GitHub page - https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects It'll be good to add it also here to have clear picture what is required, what is being worked on and what are the plans. I don't think it'll be useful all of the time, but it might help at the moment to clearly see what's what.
3. I recall there was Slack channel for QOSF. Is it still working? If not it'll be good to have some Slack or Discord (my preference)

**Comment by victor-onofre** [comment]
@s-peryt I have accepted the PR #88. Great job! The website is coming to life again!

1.  Good idea! I like the organization. I added another issue #101 
2. Yes, that is exactly what we need. I will check it
3. Yes there is a Slack channel [here](https://join.slack.com/t/qosf/shared_invite/zt-2kcd8mzio-xOhVhA286qxspgd26x6TYQ). It is active mainly for the mentorship program. @jalaln06 we are thinking of passing to Discord later.

**Comment by s-peryt** [comment]
Thanks @victor-onofre for merge. I'll work on another issues this week.

It'll be really nice to have Discord sooner than later - just a personal preference and attempt to reduce communication clutter. Which is really hard as it is :)

**Comment by s-peryt** [comment]
@victor-onofre @MaldoAlberto I've created (maybe temporary - TBD) Discord server for QOSF - https://discord.gg/9WbQTHjG . It is very much barebone at the moment. Once you'll join I can add you as moderators/admins.

**Comment by s-peryt** [comment]
@victor-onofre @MaldoAlberto any progress in putting website back online?

**Comment by s-peryt** [comment]
Thanks @victor-onofre for details.
Feel free to write issues here or on Discord. Maybe I'll be able to help, although I doubt I am an expert in GitHub static pages. I only built it once so far.
BTW, where are currently distributed details about mentorship programs?
